fc-exploitation/src/service-provider/dto/service-provider-input.dto.ts

```diff
@@ -86,8 +86,8 @@ export class ServiceProviderDto {
   readonly scopes: string[];
 
   @IsOptionalExtended()
-  @IsIn([ES256, RS256, HS256], {
-    message: `<strong>userinfo_signed_response_alg</strong> doit être une des valeurs suivantes: ${ES256}, ${RS256} ou ${HS256}`,
+  @IsIn([ES256, RS256, HS256, NONE], {
```

Comment:
```suggestion
  @IsIn([ES256, RS256, HS256, EMPTY], {
```

:question Why not using already EMPTY from AlgoValue?